resources/views/components/menu/footer.blade.php

```diff
@@ -0,0 +1,203 @@
+                'url' => url('leaderboardList.php'),
+                // 'url' => route('leaderboard.index'),
+                // 'visible' => Gate::allows('viewAny', App\Platform\Models\Leaderboard::class),
+            ],
+            // [
+            //     'active' => Route::is('system*'),
+            //     'label' => __res('system'),
+            //     'url' => route('system.index'),
+            //     'visible' => Gate::allows('viewAny', App\Platform\Models\System::class),
+            // ],
+            [
+                'active' => Route::is('game*'),
+                'label' => __res('game'),
+                'url' => url('leaderboardList.php'),
+                // 'url' => route('game.index'),
+                // 'visible' => Gate::allows('viewAny', App\Platform\Models\Game::class),
+            ],
+            [
+                'active' => Route::is('downloads*'),
+                'label' => __res('emulator'),
+                'url' => url('download.php'),
+                // 'url' => route('download.index'),
+                'visible' => true,
+            ],
+        ],
+    ],
+    [
+        'title' => __('Documentation'),
+        'items' => [
+            [
+                'label' => __('Developers Code of Conduct'),
+                'url' => 'https://docs.retroachievements.org/Developers-Code-of-Conduct',
+                'visible' => true,
+            ],
+            [
+                'label' => __('Users Code of Conduct'),
+                'url' => 'https://docs.retroachievements.org/Users-Code-of-Conduct',
+                'visible' => true,
+            ],
+            [
+                'label' => __('FAQ'),
+                'url' => 'https://docs.retroachievements.org/FAQ',
+                'visible' => true,
+            ],
+            // [
+            //     'active' => Route::is('docs.api'),
+            //     'label' => __('API'),
+            //     'url' => url('APIDemo.php'),
+            //     // 'url' => route('docs.api'),
+            //     'visible' => true,
+            // ],
+        ],
+    ],
+    [
+        'title' => __('Community'),
+        'items' => [
+            [
+                'label' => 'RANews',
+                'url' => 'https://news.retroachievements.org',
+                'visible' => true,
+            ],
+            [
+                'active' => Route::is('forum*'),
+                'label' => __res('forum'),
+                'url' => url('forum.php'),
+                // 'url' => route('forum-category.show', 1),
+                'visible' => true,
+            ],
+            [
+                'active' => Route::is('user*'),
+                'label' => __res('user'),
+                'url' => url('userList.php'),
+                // 'url' => route('user.index'),
+                'visible' => true,
+            ],
+            [
+                'active' => Route::is('news*'),
+                'label' => __res('news'),
+                'url' => url('developerstats.php'),
```

Comment:
I'm not sure what this is supposed to be pointing at, it appears to be new.

Reply:
News index is prepared but does not exist yet - linking it developerstats was a mistake.